server/emulator.go

```diff
@@ -93,14 +93,14 @@ func (m EmulatorAPIServer) CommitBlock(w http.ResponseWriter, r *http.Request) {
 }
 func (m EmulatorAPIServer) SnapshotList(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
-	badgerStore := (*m.storage).Store().(*badger.Store)
-	contexts, err := badgerStore.ListContexts()
+	snapshotProvider := (*m.storage).Store().(storage.SnapshotProvider)
```

Comment:
Do a check during type assertion since it might not be successful
```suggestion
	snapshotProvider, ok := (*m.storage).Store().(storage.SnapshotProvider)
	if !ok {}
```

Reply:
fixed  in [796ea8d](https://github.com/onflow/flow-emulator/pull/301/commits/796ea8db5ef7ec370673113de155a44638bc9d43)